src/LinAlg/hiopVectorRajaPar.cpp

```diff
@@ -1514,5 +1514,30 @@ void hiopVectorRajaPar::copyFromDev() const
   resmgr.copy(data_host, data_dev_);
 }
 
+long long hiopVectorRajaPar::numOfNegVal_w_tol(const double &tol) const
+{  
+  double* data = data_dev_;
+  RAJA::ReduceSum<hiop_raja_reduce, int> sum(0);
```

Comment:
The function returns a `long long`, but the reduction variable uses an `int` - I think these should be consistent.